lib/browser/api/session.js

```diff
@@ -35,6 +35,7 @@ Object.defineProperty(exports, 'defaultSession', {
 const wrapSession = function (session) {
   // Session is an EventEmitter.
   Object.setPrototypeOf(session, EventEmitter.prototype)
+  process.emit('session-created', session)
```

Comment:
How about making this event part of `session` module, so users can listen to it by:

```js
const {session} = require('electron')
session.on('session-created', () => {
})
```